packages/core/src/browser/style/sidepanel.css

```diff
@@ -86,7 +86,7 @@
 }
 
 .p-TabBar.theia-app-left .p-TabBar-tab.p-mod-current.theia-mod-active {
-    border-left: var(--theia-panel-border-width) solid var(--theia-focusBorder);
+    border-left: var(--theia-panel-border-width) solid var(--theia-list-highlightForeground);
```

Comment:
it looks weird semantically, list color variables should be applied to lists or trees, not to tabs

Reply:
>  List/Tree foreground color of the match highlights when searching inside the list/tree.

from https://code.visualstudio.com/api/references/theme-color#lists-and-trees

I don't think it is a proper variable to be used here.